lib/ansible/module_utils/basic.py

```diff
@@ -1879,21 +1926,11 @@ def append_to_file(self, filename, str):
         fh.write(str)
         fh.close()
 
-    def pretty_bytes(self,size):
-        ranges = (
-                (1<<70, 'ZB'),
-                (1<<60, 'EB'),
-                (1<<50, 'PB'),
-                (1<<40, 'TB'),
-                (1<<30, 'GB'),
-                (1<<20, 'MB'),
-                (1<<10, 'KB'),
-                (1, 'Bytes')
-            )
-        for limit, suffix in ranges:
-            if size >= limit:
-                break
-        return '%.2f %s' % (float(size)/ limit, suffix)
+    def pretty_bytes(self, size, isbits=False, unit=None):
+        return pretty_bytes(size, isbits, unit)
+
+    def string_to_bytes(self, size, default_unit = None):
+        return string_to_bytes(size, default_unit)
```

Comment:
I don't have a problem with _check_type_bytes simply calling the toplevel string_to_bytes(); no need for this method.  (Wish we could do the same with pretty_bytes() but backwards compat... )